com.salesforce.b2eclipse.jdt.ls/src/main/java/com/salesforce/b2eclipse/command/internal/BazelQueryHelper.java

```diff
@@ -159,14 +159,24 @@ public BazelQueryHelper(BazelCommandExecutor bazelCommandExecutor) {
     public synchronized List<String> getJavaPackages(File bazelWorkspaceRootDirectory, WorkProgressMonitor progressMonitor) throws IOException, InterruptedException, BazelCommandLineToolConfigurationException {
         
         ImmutableList.Builder<String> argBuilder = ImmutableList.builder();
+        final String osName = System.getProperty("os.name");
+        StringBuilder javaProjectIndicators = new StringBuilder();
+        javaProjectIndicators.append("kind('");
+        javaProjectIndicators.append(String.join("|", BazelBuildFileHelper.JAVA_PROJECT_INDICATORS));
+        javaProjectIndicators.append("', //...)");
         
         argBuilder.add("query");
-        argBuilder.add("kind(\"" + String.join("|", BazelBuildFileHelper.JAVA_PROJECT_INDICATORS) + "\", //...)");
+        // 3 types of behavior: 1. Windows (command line and java.lang.Process), 2. non-Windows command line, 3. non-Windows java.lang.Process
+        if (osName != null && osName.toLowerCase().startsWith("win")) {
```

Comment:
since we have Commons Lang in the classpath, I would use SystemUtils.IS_OS_WINDOWS to determine OS

Reply:
replaced as proposed